gateway/Dockerfile.in

```diff
@@ -1,13 +1,6 @@
-FROM debian:9.5
-
-RUN apt-get update -y && \
-    apt-get install -y nginx && \
-    rm -rf /var/lib/apt/lists/*
+FROM nginx:1.15-alpine
```

Comment:
I don't prefer this at this point.  Alpine is much harder to work with if you're trying to debug anything (and we're at the stage where we often are.)  Thoughts, @danking?

Reply:
Why is it harder to debug? To debug Nginx requests I view the log, where error.log and access.log are symlinked to stderr/stdout.

Reply:
Sometimes you want to log in and do stuff and Alpine has none of the tooling that makes that easy (which is why it is so small).

Reply:
I can remove the change if there is something that we're doing that alpine can't cover. The other motivation for this change was to move us to Nginx 1.15.N from 1.10.N, which is years old.

Reply:
I'm not talking about the normal behavior, I'm talking about the developer experience when you log into the pod to muck around, e.g. you get bash instead of sh.  I'm just trying to make things easy for us.  I don't think image size matters here, and if all the images are based on Debian, it will be cached on every node and there is very little overhead.  I'd even prefer a standard image build on Debian with useful stuff installed (e.g. emacs, which I immediately install every time I log into a pod).

I'm all for upgrading.  You can do that without switching to Alpine.  Let's make that a separate PR.

Reply:
Ok, I'll roll back the change.